Tanvir04Khan/01-max-udemy/03-login-page/src/App.js

```diff
@@ -1,17 +1,27 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 
-import Header from "./Components/Login/Header";
+import LoginPage from "./Components/Login/LoginPage";
+import HomePage from "./Components/Home/HomePage";
 
 import "./App.css";
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
 
+  useEffect(() => {
+    if (localStorage.isLoggedIn === "1") {
+      setIsLoggedIn(true);
+    }
+  }, []);
+
+  const isloggedin = (boolean) => {
+    setIsLoggedIn(boolean);
+  };
```

Comment:
I dont get it 

isLoggedin (capital L) vs isloggedin (small)

Reply:
also, why are you using a datatype as a variable name?

Reply:
changed function name and variable name @faran4engg  @AlishaPrasad